sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -531,6 +531,7 @@ def test_value_provider_transform(self):
            | "WriteWithMultipleDests2" >> beam.io.gcp.bigquery.WriteToBigQuery(
                table=value_provider.StaticValueProvider(
                    str, '%s:%s' % (self.project, output_table_2)),
+               schema=bigquery.SCHEMA_AUTODETECT,
```

Comment:
Note that SCHEMA_AUTODETECT if BQ has to ingest `bytes` data into a table that has `bytes` field.
Also, should we have a test with and without autodetect behavior?

Reply:
There are tests with and without this feature running in that file. WDYT?

Reply:
I think that should be fine and we don't need to duplicate every test twice, but you might want to do a one-off run of integration tests where the value of autodetection flag is changed to the opposite value, to make sure there are no surprises.